test/parallel/test-debug-prompt.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common');
+const spawn = require('child_process').spawn;
+
+const timeoutId = setTimeout(function() {
```

Comment:
I suppose it is redundant. I was just modeling off of [another test](https://github.com/nodejs/node/blob/master/test/parallel/test-force-repl.js) that spawns a node process.